src/rt/tracegc.d

```diff
@@ -0,0 +1,545 @@
+/**
+ * Contains implementations of functions called when the
+ *   -profile=gc
+ * switch is thrown.
+ *
+ * Copyright: Copyright Digital Mars 2015 - 2015.
+ * License: Distributed under the
+ *      $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0).
+ *    (See accompanying file LICENSE)
+ * Authors:   Walter Bright
+ * Source: $(DRUNTIMESRC src/rt/_tracegc.d)
+ */
+
+module rt.tracegc;
+
+//version = tracegc;
+
+import rt.profilegc;
+
+version (tracegc) import core.stdc.stdio;
+
+version (none)
```

Comment:
I copied this code from druntime with no changes. I presume that reason for `none` is that is not useful as automatic test case and only intended for manual copying to check compiler generated code.